js/components/portal-dashboard/num-students-container.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import css from "../../../css/portal-dashboard/num-students.less";
+
+  interface IProps {
+    studentCount: number;
+  }
+
+export class NumberOfStudentsContainer extends React.PureComponent<IProps> {
+    render() {
+      return (
+        <div className={css.numStudentsContainer} data-test="num-student-container">
+          <span className={css.containerLabel}>Class: </span>
+          <span data-test="student-number">{this.props.studentCount}</span>
+          <span> students</span>
+        </div>
+      );
+    }
+  }
```

Comment:
Add newline at end of file.

Reply:
This should get caught by linter. One thing we could do is add a git hook so you can't push without a clean lint, for instance.